storage/range.go

```diff
@@ -1000,7 +1000,7 @@ func (r *Range) maybeGossipFirstRange() error {
 		log.Errorc(ctx, "failed to gossip cluster ID: %s", err)
 	}
 
-	if ok, err := r.getLeaseForGossip(ctx); !ok || err != nil {
+	if ok, err := r.getLeaseForGossip(ctx); !ok {
```

Comment:
The general convention is that if `err != nil`, the other return values are undefined. I'd rather keep the `err != nil` check rather than rely on getLeaseForGossip to always return false when there is an error.